### PR TITLE
Fix duplicate bug for add feature

### LINKED ISSUE #75: Prevent duplicate client/property entries for Add feature

### PR DESCRIPTION
**Pull request by OVReader**
Fix #75 
Duplicate client/property can no longer be added to list.
Same client refers to having the same name/contact number/email.
Same property refers to having the same address.

### CODE REVIEW

File: src/main/java/seedu/duke/exception/DuplicateClientException.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package seedu.duke.exception;
+
+import static seedu.duke.Messages.MESSAGE_DUPLICATE_CLIENT;
+
+//@@author OVReader
```
Comment (wilsonngja) [inline src/main/java/seedu/duke/exception/DuplicateClientException.java:5]: I think @@author is supported in '/*' format.
Can consider adding inside the header.

File: src/main/java/seedu/duke/exception/InvalidBudgetFormatException.java [hunk 2]
```diff
@@ -1,18 +1,14 @@
...
-    private final String message;
-
-    public InvalidBudgetFormatException(String message) {
-        this.message = message;
-    }
-
     @Override
     public String toString() {
-        return message;
+        return MESSAGE_INVALID_BUDGET_FORMAT;
```
Comment (wilsonngja) [inline src/main/java/seedu/duke/exception/InvalidBudgetFormatException.java:12]: Good use of constant with meaningful name.

**Review by wilsonngja — COMMENTED** [review]
Good coding standard. I see that some of the documentation errors has been fixed. Good job and it LGTM.

**Review by FeliciaBeatrice — APPROVED** [review]
everything looks good! LGTM

**Pull request merged**